conans/client/tools/win.py

```diff
@@ -490,3 +490,32 @@ def run_in_windows_bash(conanfile, bashcmd, cwd=None, subsystem=None, msys_mingw
         conanfile.output.info('run_in_windows_bash: %s' % wincmd)
         # https://github.com/conan-io/conan/issues/2839 (subprocess=True)
         return conanfile._conan_runner(wincmd, output=conanfile.output, subprocess=True)
+
+
+def replace_windows_paths_in_file(file_path, path_search, replace, strict=True):
```

Comment:
I'm thinking about use the same "replace_in_file" but with `case_sensitive=True` and `pathsep_sensitive=True`